build-support/bin/generate_travis_yml.py

```diff
@@ -589,16 +589,13 @@ def rust_tests_osx() -> Dict:
     **_RUST_TESTS_BASE,
     "name": "Rust tests - OSX",
     "os": "osx",
-    "osx_image": "xcode11",
-    # We cannot currently run fuse tests in CI, as the osx_image doesn't support it.
-    # Until that changes, no need to install osxfuse.
-    # "addons": {
-    #   "homebrew": {
-    #     "casks": ["osxfuse"]
-    #   }
-    # },
+    # We need to use xcode8.3 because newer versions of OSX won't let new kexts be installed
+    # without travis taking some action, and we need the osxfuse kext.
+    # See https://github.com/travis-ci/travis-ci/issues/10017
+    "osx_image": "xcode8.3",
     "before_install": [
       './build-support/bin/install_python_for_ci.sh "${PYENV_PY36_VERSION}"',
+      "brew cask install osxfuse",
```

Comment:
It's important to have an equally lucid comment explaining why we do this and not use the standard travis "addons" thing.